src/app/layout.tsx

```diff
@@ -58,7 +63,9 @@ export default async function RootLayout({
         data-ga4-measurement-id={process.env.NEXT_PUBLIC_GA4_MEASUREMENT_ID}
         data-node-env={process.env.NODE_ENV}
       >
-        <SessionProvider session={session}>{children}</SessionProvider>
+        <PublicEnvProvider publicEnv={PUBLIC_ENV}>
```

Comment:
A provider for public env variables so we can access them from client components.